package.json

```diff
@@ -54,8 +54,8 @@
   },
   "homepage": "https://github.com/mozilla/network-pulse#readme",
   "devDependencies": {
-    "chokidar": "^2.1.5",
-    "chokidar-cli": "^1.2.2",
+    "chokidar": "^3.3.0",
```

Comment:
I don't see any breaking updates except [this one](https://github.com/paulmillr/chokidar/blob/master/.github/full_changelog.md#chokidar-310-sep-16-2019)
> Breaking: dotfiles are no longer filtered out by default. Use ignored option if needed.

But we don't have any dotfiles in the repo so no action needed on our end.